db/model/subject.js

```diff
@@ -262,23 +222,21 @@ module.exports = function subject(seq, dataTypes) {
        *  record
        */
       afterCreate(inst /* , opts */) {
+        /*
+         * add entry to the subjectStore in redis only if the subject
+         * is published and the sampleStoreFeature is enabled
```

Comment:
really? don't *all* subjects get added to sample store when sampleStoreFeature is enabled, even unpublished ones?

Reply:
I did not add this comment; but yes, we are adding subjects to the sampleStore only when it is published. The aspects are added no matter what. I have a note to create stories to fix issues like this.